BlockchainSdk/Blockchains/TON/TONWalletManager.swift

```diff
@@ -204,4 +213,14 @@ private extension TONWalletManager {
             return Fee(amount, parameters: fee.parameters)
         }
     }
+
+    private func createExpirationTimestampSecs() -> UInt32 {
+        UInt32(Date().addingTimeInterval(TimeInterval(Constants.transactionLifetimeInMin * 60)).timeIntervalSince1970)
+    }
+}
+
+private extension TONWalletManager {
+    enum Constants {
+        static let transactionLifetimeInMin: Double = 1
```

Comment:
может лучше сразу в sec здесь сделать? Что бы выше не умножать просто
```suggestion
        static let transactionLifetimeInSec: TimeInterval = 60
```